schemas/ingest-spans.v1.schema.json

```
@@ -0,0 +1,78 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "span_kafka_message",
+  "properties": {
+    "event_id": {
+      "type": "string"
+    },
+    "organization_id": {
+      "type": "integer"
+    },
+    "project_id": {
+      "type": "integer"
+    },
+    "retention_days": {
+      "type": "integer"
+    },
+    "span": {
+      "$ref": "#/definitions/Span"
+    }
+  },
+  "required": ["organization_id", "project_id", "retention_days", "span"],
+  "definitions": {
+    "Span": {
+      "type": "object",
+      "title": "span",
+      "additionalProperties": true,
+      "properties": {
+        "description": {
+          "type": "string"
+        },
+        "exclusive_time_ms": {
+          "type": "integer"
+        },
```

```suggestion
        "exclusive_time": {
          "type": "number"
        },
```

See https://github.com/getsentry/relay/blob/bd15ccff76d893117ab6a56b7e0097002733b2c2/tests/integration/test_store.py#L1242